lms/djangoapps/verify_student/migrations/0005_auto__add_incoursereverificationconfiguration.py

```diff
@@ -97,7 +97,7 @@ def backwards(self, orm):
         },
         'verify_student.verificationcheckpoint': {
             'Meta': {'unique_together': "(('course_id', 'checkpoint_name'),)", 'object_name': 'VerificationCheckpoint'},
-            'checkpoint_name': ('django.db.models.fields.CharField', [], {'unique': 'True', 'max_length': '32'}),
```

Comment:
@awais786 it looks like this migration would have already been executed on prod.  can you confirm that this change doesn't require anything to be altered in the database?